sdk/src/main/java/io/dapr/client/DaprClientGrpc.java

```diff
@@ -677,6 +683,133 @@ public void close() throws Exception {
     ).then();
   }
 
+  /**
+   *{@inheritDoc}
+   */
+  @Override
+  public Mono<ConfigurationItem> getConfiguration(GetConfigurationRequest request) {
+    try {
+      final String configurationStoreName = request.getStoreName();
+      final String key = request.getKey();
+      final Map<String, String> metadata = request.getMetadata();
+
+      if ((configurationStoreName == null) || (configurationStoreName.trim().isEmpty())) {
+        throw new IllegalArgumentException("Configuration Store Name cannot be null or empty.");
+      }
+      if ((key == null) || (key.trim().isEmpty())) {
+        throw new IllegalArgumentException("Key cannot be null or empty.");
+      }
+      DaprProtos.GetConfigurationRequest.Builder builder = DaprProtos.GetConfigurationRequest.newBuilder()
+          .setStoreName(configurationStoreName)
+          .addKeys(key);
+      if (metadata != null) {
+        builder.putAllMetadata(metadata);
+      }
+
+      DaprProtos.GetConfigurationRequest envelope = builder.build();
+      return this.getConfigurationAlpha1(envelope).map(
+          it -> it.get(0)
+      );
+    } catch (Exception ex) {
+      return DaprException.wrapMono(ex);
+    }
+  }
+
+  /**
+   * {@inheritDoc}
+   */
+  @Override
+  public Mono<List<ConfigurationItem>> getConfigurations(GetBulkConfigurationRequest request) {
```

Comment:
Rename GetBulkConfigurationRequest to GetConfigurationRequest
Rename method to getConfiguration

The other methods reduce to this single call.